central/securedclustercertgen/certificates.go

```diff
@@ -61,8 +84,8 @@ func IssueLocalScannerCerts(namespace string, clusterID string) (*storage.TypedS
 	return &certsSet, nil
 }
 
-func localScannerCertificatesFor(serviceType storage.ServiceType, namespace string, clusterID string) (caPem []byte, cert *storage.TypedServiceCertificate, err error) {
-	certificates, err := generateServiceCertMap(serviceType, namespace, clusterID)
+func (c *certIssuerImpl) certificatesFor(serviceType storage.ServiceType, namespace string, clusterID string) (caPem []byte, cert *storage.TypedServiceCertificate, err error) {
```

Comment:
```suggestion
func (c *certIssuerImpl) certificateFor(serviceType storage.ServiceType, namespace string, clusterID string) (caPem []byte, cert *storage.TypedServiceCertificate, err error) {
```

Reply:
It's really just returning a cert for a single service, hence I think singular would be a better fit (yes, I know it is also returning the CA cert, but that's a different thing in my opinion).